honeybee_schema/energy/properties.py

```diff
@@ -105,6 +106,15 @@ class FaceEnergyPropertiesAbridged(NoExtraBaseModel):
         'Model global_construction_set.'
     )
 
+    vent_crack: AFNCrack = Field(
+        default=None,
+        description='An optional AFNCrack to specify airflow through a surface crack '
+        'used by the AirflowNetwork. If this value is None and one of the '
+        'AirflowNetwork options are selected in the VentilationSimulationControl '
+        'object, a default value will be calculated to produce air flow leakages '
+        'equivalent to the zone flow rate defined by the corresponding Infiltration object.'
```

Comment:
I think you are confusing the job of the schema and the core Python libraries here. By the time the model has gotten to be in a schema format, it should not have a `None` value here if the user intends to use the AFN. Instead, if this value is `None`, I think this should mean that there's just no crack for the Face in the resulting AirFlowNetwork. So it's like there's no leakage from the wall at all.

We will have methods in the core library that can compute this property from the Room's infiltration (overwriting whatever is here). But we should have the user explicitly call this method if they want it and we shouldn't assume that they always want to auto-compute it if they leave this as None.

Reply:
You're right, I did get that confused. Fixed.